### DISCUSSION TITLE
/why-i-stopped-using-intel-gvt-g-on-proxmox/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#173] [Announcements]
Why I stopped using Intel GVT-g on Proxmox with Quick Sync
        
[https://blog.ktz.me/why-i-stopped-using-intel-gvt-g-on-proxmox/](https://blog.ktz.me/why-i-stopped-using-intel-gvt-g-on-proxmox/)

**Comment by tappyson** [comment]
Interesting read... when you say on the host itself do you mean through LXC's or the actual host? Have you done any comparisons of LXC - VM - HOST?

**Reply by ironicbadger** [reply]
I haven't actually. I'm much more into the docker scene with compose so tbh never really used LXCs until recently when I wanted to run Pihole in an encapsulated system with its own systemd instance available.

**Reply by ironicbadger** [reply]
Host means bare metal host.

**Comment by eyyit** [comment]
Can you map the igpu directly to the vm/container instead of virtualizing it similar to vfio etc?

**Reply by ironicbadger** [reply]
Technically, yes. That is called passthrough. It's even more unreliable, though.

**Reply by eyyit** [reply]
Can you volume mount `/dev/dri` into the container from proxmox for quicksync usage?

**Reply by ironicbadger** [reply]
You must be explicit in your question. Are you discussing VMs? Containers (as in docker)? or LXCs? The answer is different for all 3.

**Reply by eyyit** [reply]
Sorry about that.  It is my understanding that you can access everything you need from quicksync via `/dev/dri`.  Currently I am volume mounting this from my linux os into a docker container running plex which seems to enable hw transcode just fine.  I'm curious what your take on this is, and if this is effective, could you not then do the same thing from proxmox?  Volume mount `/dev/dri` into a container running on proxmox (docker or otherwise)?  I am trying to decide if I want to migrate everything from linuxOS that runs docker containers to proxmox running those same containers plus a few VMs under qemu.

I currently run 2 qemu vms and about 10 docker containers, but proxmox seems like a nice system to facilitate all this.  I just don't want to dive in with multiple days worth of work only to find that something like quicksync instability means I'll have to undo it all.

Thanks!

**Reply by ironicbadger** [reply]
Running any quicksync dependent service in a VM has been a poor result for me. I now run Plex / Jellyfin on the host as containers with `/dev/dri` mounted. Works great.

**Comment by IAmSHOCK** [comment]
Hi, when you say you're using Plex on bare metal, does it mean your not using proxmox anymore? How did you install Plex?

**Reply by ironicbadger** [reply]
https://perfectmediaserver.com/01-overview/high-level/

**Comment by casperse** [comment]
I am using Unraid and my GPU and IGPU is shared between many Dockers? How is this accomplished? Isn’t it using the same technical solution in Linux? (sorry I know this is an older post but I would like to know I’d this is still a thing? Looking at using Proxmox and I like this functionality in Unraid :-)

**Comment by kimsonvu** [comment]
You are right about Stability. I test it with the mainboard asrock E3C246D2I  with Xeon E 2144G. My Proxmox kernel panics show it after only 12 hours. First is Windows 11 panics and after that everything. I quit and move to pass through gpu and igpu into 2 VM.
